src/server/schemas/commonTxBody.ts

```diff
@@ -0,0 +1,10 @@
+import { Type } from "@sinclair/typebox";
+
+export const commonTxBodySchema = Type.Object({
+  externalMetadata: Type.Optional(
+    Type.String({
+      description:
+        "External metadata that is returned to webhook listeners. If used for JSON, we recommend base64 encoding a stringified JSON object.",
+    }),
+  ),
+});
```

Comment:
can we move this `sharedApiSchema.ts`. We already a file which has shared types/interfaces

Reply:
`External metadata that is returned to webhook listeners.`, not only on webhooks but also, on the status end-point